personalize.module

```diff
@@ -1100,6 +1100,17 @@ function personalize_agent_set_status($agent_name, $status) {
     $end_time_var = _personalize_agent_get_stoptime_variable($agent_name);
     $now = time();
     variable_set($end_time_var, $now);
+    // Check if we need to delete all the actions for completed campaigns.
+    if (variable_get('personalize_delete_actions_on_campaign_completion', FALSE)) {
```

Comment:
The plan is to eventually expose these as config settings - I just didn't want to introduce that yet so that I could get it in under the radar, i.e. "no UI change here, nothing to see, move along!" :P